pandas/indexes/multi.py

```diff
@@ -1499,6 +1499,24 @@ def reorder_levels(self, order):
     def __getslice__(self, i, j):
         return self.__getitem__(slice(i, j))
 
+    def _lexsort_indexer(self, orders=True, na_position='last'):
+        from pandas.core.sorting import lexsort_indexer
+        from pandas.core.categorical import Categorical
+        from pandas.core.algorithms import unique
+
+        # if not self.is_lexsorted():
+        #     mi = MultiIndex.from_tuples(self.values)
+        # else:
+        #     mi = self
+
+        keys = []
```

Comment:
rename to ``_get_labels_for_sorting()``

add a doc-string.

rename ``keys`` -> ``labels``